### DISCUSSION TITLE
Issue with creating generic List or Dictionary in GlobalParams

### DISCUSSION THREAD

**Discussion by lakhanrathinice** [microsoft/RulesEngine#375] [Q&A]
Hello,

For one of the requirement I am trying to create generic List in GlobalParams using expression but getting error as "Type 'List' not found". I am getting similar error for generic Dictionary as well. I have tried declaring List as custom type in reSettings but getting same error.

Please let me know if someone has any idea on this.

![image](https://user-images.githubusercontent.com/98881427/180286905-9e61cf36-24ba-4587-bdc5-10589b99fbeb.png)

**Comment by djl-85** [comment]
Did you fix your problem? I am having same issue. Thank you.
